pipeline/transformer/transformer_marc_common/src/main/scala/weco/pipeline/transformer/marc_common/transformers/MarcNotes.scala

```diff
@@ -0,0 +1,132 @@
+package weco.pipeline.transformer.marc_common.transformers
+import grizzled.slf4j.Logging
+import weco.catalogue.internal_model.work.{Note, NoteType}
+import weco.pipeline.transformer.marc_common.models.{
+  MarcField,
+  MarcRecord,
+  MarcSubfield
+}
+import weco.pipeline.transformer.text.TextNormalisation._
+
+import java.net.URL
+import scala.util.Try
+trait MarcNotes extends Logging {
+
+  protected val globallySuppressedSubfields: Set[String] = Set("5")
+
+  val notesFields: Map[String, MarcField => Note] = Map(
+    "500" -> createNoteFromContents(NoteType.GeneralNote),
+    "501" -> createNoteFromContents(NoteType.GeneralNote),
+    "502" -> createNoteFromContents(NoteType.DissertationNote),
+    "504" -> createNoteFromContents(NoteType.BibliographicalInformation),
+    "505" -> createNoteFromContents(NoteType.ContentsNote),
+    "506" -> createNoteFromContents(NoteType.TermsOfUse),
+    "508" -> createNoteFromContents(NoteType.CreditsNote),
+    "510" -> createNoteFromContents(NoteType.ReferencesNote),
+    "511" -> createNoteFromContents(NoteType.CreditsNote),
+    "514" -> createNoteFromContents(NoteType.LetteringNote),
+    "515" -> createNoteFromContents(NoteType.NumberingNote),
+    "518" -> createNoteFromContents(NoteType.TimeAndPlaceNote),
+    "524" -> createNoteFromContents(NoteType.CiteAsNote),
+    "525" -> createNoteFromContents(NoteType.GeneralNote),
+    "533" -> createNoteFromContents(NoteType.ReproductionNote),
+    "534" -> createNoteFromContents(NoteType.ReproductionNote),
+    "535" -> createLocationOfNote,
+    "536" -> createNoteFromContents(NoteType.FundingInformation),
+    "540" -> createNoteFromContents(NoteType.TermsOfUse),
+    "542" -> createNoteFromContents(NoteType.CopyrightNote),
+    "544" -> createNoteFromContents(NoteType.RelatedMaterial),
+    "545" -> createNoteFromContents(NoteType.BiographicalNote),
+    "546" -> createNoteFromContents(NoteType.LanguageNote),
+    "547" -> createNoteFromContents(NoteType.GeneralNote),
+    "550" -> createNoteFromContents(NoteType.GeneralNote),
+    "525" -> createNoteFromContents(NoteType.GeneralNote),
+    "561" -> createOwnershipNote,
+    "562" -> createNoteFromContents(NoteType.GeneralNote),
+    "563" -> createNoteFromContents(NoteType.BindingInformation),
+    "580" -> createNoteFromContents(NoteType.GeneralNote),
+    "581" -> createNoteFromContents(NoteType.PublicationsNote),
+    "585" -> createNoteFromContents(NoteType.ExhibitionsNote),
+    "586" -> createNoteFromContents(NoteType.AwardsNote),
+    "588" -> createNoteFromContents(NoteType.GeneralNote)
+  )
+  def toNotes(
```

Comment:
I didn't like the way apply worked before, with the map creating field/transformer pairs due to the need to check a few conditions beforehand.

I think this is neater.